test/workload.yaml

```diff
@@ -10,8 +10,12 @@
   spec:
     interface: eth0
     profiles: [prof1, prof2]
-    ipNetworks: [1.2.3.4/32, "00:bb::aa/10"]
+    ipNetworks: ["1.2.3.4/32", "00:bb::aa/128"]
+    ipNATs:
+    - externalIP: 10.20.30.40
```

Comment:
current 4 person poll (me and you included) prefer internal/external (3:1).  I suggest leaving as is, and we can rename later if we need to.